src/test/java/com/google/api/codegen/testdata/php/php_library.baseline

```diff
@@ -461,8 +467,8 @@ class LibraryServiceGapicClient
      *           The transport used for executing network requests. May be either the string `rest`
      *           or `grpc`. Defaults to `grpc` if gRPC support is detected on the system.
      *           *Advanced usage*: Additionally, it is possible to pass in an already instantiated
-     *           TransportInterface object. Note that when this objects is provided, any settings in
-     *           $transportConfig, and any $serviceAddress setting, will be ignored.
+     *           {@see TransportInterface} object. Note that when this object is provided, any
```

Comment:
There is an odd restriction that we don't have a workaround for in the doc generation yet. I believe the `@see` tag needs to include a FQNS in order to build out a link, if I remember correctly it doesn't take into account use statements.